doc/classes/WorkerThreadPool.xml

```diff
@@ -1,10 +1,50 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 <class name="WorkerThreadPool" inherits="Object" version="4.1" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="../class.xsd">
 	<brief_description>
+		A singleton that allocates some [Thread]s on startup, used to offload tasks to these threads.
 	</brief_description>
 	<description>
+		The [WorkerThreadPool] singleton allocates a set of [Thread]s (called worker threads) on project startup and provides methods for offloading tasks to them. This can be used for simple multithreading without having to create [Thread]s.
+		Tasks hold the [Callable] to be run by the threads. [WorkerThreadPool] can be used to create regular tasks, which will be taken by one worker thread, or group tasks, which can be distributed between multiple worker threads. Group tasks execute the [Callable] multiple times, which makes them useful for iterating over a lot of elements, such as the enemies in an arena.
+		Here's a sample on how to offload an expensive function to worker threads:
+		[codeblocks]
+		[gdscript]
+		var enemies = [] # An array to be filled with enemies.
+
+		func process_enemy_ai(enemy_index):
+		    var processed_enemy = enemies[enemy_index]
+		    # Expensive logic...
+
+		func _process(delta):
+		    var task_id = WorkerThreadPool.add_group_task(process_enemy_ai, enemies.size())
+		    # Other code...
+		    WorkerThreadPool.wait_for_group_task_completion(task_id)
+		    # Other code that depends on the enemy AI already being processed.
+		[/gdscript]
+		[csharp]
+		private List&lt;Node&gt; _enemies = new List&lt;Node&gt;(); // A list to be filled with enemies.
+
+		private void ProcessEnemyAI(int enemyIndex)
+		{
+		    Node processedEnemy = enemies[enemyIndex];
+		    // Expensive logic here.
+		}
+
+		public override void _Process(double delta)
+		{
+		    long taskID = WorkerThreadPool.AddGroupTask(Callable.From&lt;int&gt;(ProcessEnemyAI), enemies.Length);
```

Comment:
Since `_enemies` is now a list, it should be using the `Count` property. Also noticed the missing underscore (`_`).

Also renamed `taskId` to follow .NET naming convention (ID is one of the [special cases](https://learn.microsoft.com/en-us/dotnet/standard/design-guidelines/capitalization-conventions#capitalizing-compound-words-and-common-terms)). Sorry for not noticing earlier.

```suggestion
		    long taskId = WorkerThreadPool.AddGroupTask(Callable.From&lt;int&gt;(ProcessEnemyAI), _enemies.Count);
```